Gemfile.lock

```diff
@@ -133,18 +133,6 @@ PATH
     veteran_confirmation (0.0.1)
     vye (0.1.0)
 
-GEM
-  remote: https://enterprise.contribsys.com/
-  specs:
-    sidekiq-ent (7.2.4)
-      einhorn (~> 1.0)
-      gserver
-      sidekiq (>= 7.2.0, < 8)
-      sidekiq-pro (>= 7.2.0, < 8)
-    sidekiq-pro (7.2.1)
-      base64
-      sidekiq (>= 7.2.0, < 8)
-
```

Comment:
is there a specific reason for the changes to the gemfile.lock?

Reply:
I do not know how or why that got deleted. I am putting it back.